classes/Tools.php

```diff
@@ -3691,15 +3790,17 @@ public static function arrayUnique($array)
      */
     public static function cleanNonUnicodeSupport($pattern)
     {
-        @trigger_error(
-            sprintf(
-                '%s is deprecated since version 8.0.0. Its use is not required.',
-                __METHOD__
-            ),
-            E_USER_DEPRECATED
-        );
+        $context = Context::getContext();
```

Comment:
`cleanNonUnicodeSupport` has been deprecated in this PR https://github.com/PrestaShop/PrestaShop/pull/27657

Reply:
sorry for that